packages/expo-doctor/src/__mocks__/@expo/spawn-async.ts

```diff
@@ -0,0 +1,12 @@
+import { SpawnResult } from '@expo/spawn-async';
```

Comment:
we usually put mocks in the root https://github.com/expo/expo/tree/main/packages/%40expo/cli/__mocks__